tests/auth.rs

```diff
@@ -183,37 +134,22 @@ fn auth_multiple_accounts_pass(
         assert!(parsed.find(Text).any(|x| x.text() == file));
     }
 
-    child.kill()?;
-
     Ok(())
 }
 
 #[rstest]
-fn auth_multiple_accounts_wrong_username(tmpdir: TempDir, port: u16) -> Result<(), Error> {
-    let mut child = register_accounts(
-        Command::cargo_bin("miniserve")?
-            .arg(tmpdir.path())
-            .arg("-p")
-            .arg(port.to_string())
-            .stdout(Stdio::null())
-            .stderr(Stdio::null()),
-    )
-    .spawn()?;
-
-    sleep(Duration::from_secs(1));
-
+fn auth_multiple_accounts_wrong_username() -> Result<(), Error> {
+    let server = server_no_stderr(ACCOUNTS);
```

Comment:
You ought to be able to do this via partial injection as linked before. There are more occurrences of this.